### ISSUE TITLE
Miss leading permissions diff when user or API key is member of a team 

### ISSUE THREAD

**Issue by dsecik** [ns1-terraform/terraform-provider-ns1#197]
Same problem as described in #15 and fixed in #100 for most of the permissions is still present for dns_zones_allow and  dns_zones_deny. Could the diff for these be also suppressed if user or API key is member of a team same as for other permissions?

### Terraform Version
Terraform: v1.0.11
terraform-provider-ns1: v1.12.2

### Affected Resource(s)
Please list the resources as a list, for example:
- ns1_user
- ns1_apikey

### Terraform Configuration Files
```hcl
resource "ns1_team" "team" {
  name            = "test-team"
  dns_view_zones  = true
  dns_zones_allow = ["some-zone"]
}

resource "ns1_user" "user" {
  name = "Test User"
  username = "test-user"
  email = "test-user@example.com"
  notify = {
    billing = false
  }
  teams = [ns1_team.team.id]
}
```

### Debug Output
N/A

### Panic Output
N/A

### Expected Behavior
After resources are created, diff should be empty on subsequent  terraform runs.
### Actual Behavior
Diff not empty, Terraform wants to remove the allowed zone inherited from the team.

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. `terraform apply`
2. `terraform plan` or `terraform apply`

### Important Factoids
N/A

### References
- #15 
- #100

**Comment by eravin-ns1** [comment]
Apologies for the long delay in getting to this. Fixed in pre-release 1.13.4-pre1, please let us know if there are any problems.

**Comment by eravin-ns1** [comment]
Fixed in normal release 1.13.4.
